search/tweets_engine.go

```diff
@@ -86,14 +75,15 @@ func (e *ESEngine) CombineResultsToSingleHit(resultsByLang map[string]*elastic.S
 		}
 
 		hit := &elastic.SearchHit{
+			Source:    result.Hits.Hits[0].Source,
 			Type:      hitType,
-			Score:     &maxScore,
+			Score:     result.Hits.Hits[0].Score,
```

Comment:
Why Hits[0].Score? Is it max score?

What is no hits? it will generate an error, test for existance. Also score might not exist it is a pointer...

Reply:
What do you do if it is not set?